### PR TITLE
feat(log-viewer): added string element with current/match support

### PR DESCRIPTION
**Pull request by mcoker**
fixes https://github.com/patternfly/patternfly/issues/4163

https://patternfly-pr-4166.surge.sh/components/log-viewer

### CODE REVIEW

**Review by nicolethoen — COMMENTED** [review]
LGTM! only question: @mcarrano, do you think the colors are okay now that they are at a smaller scale?

**Comment by mcarrano** [comment]
@nicolethoen Yes, I do this this looks fine.  One thing that occurred to me, however, is that I'm not sure what this will look like over dark theme. @mcoker can we see an example of that?

**Comment by mcoker** [comment]
@mcarrano @nicolethoen updated to universally light/dark theme compatible colors, courtesy of @mceledonia

**Review by nicolethoen — APPROVED** [review]

**Review by mceledonia — APPROVED** [review]

File: src/patternfly/components/LogViewer/examples/LogViewer.md [hunk 1]
```diff
@@ -118,10 +118,11 @@ import './LogViewer.css';
...
 | `.pf-c-log-viewer__text` | `<div>` | Initiates a log viewer text element. **Required** |
+| `.pf-c-log-viewer__string` | `<span>` | Initiates a log viewer string element. |
 | `.pf-c-log-viewer__timestamp` | `<div>` | Initiates a log viewer text element. **Required** |
 | `.pf-m-wrap-text` | `.pf-c-log-viewer` | Modifies the log viewer text to wrap. |
 | `.pf-m-line-numbers` | `.pf-c-log-viewer` | Modifies the log viewer to display line numbers. |
 | `.pf-m-dark` | `.pf-c-log-viewer` | Modifies the log viewer content for dark theme. |
-| `.pf-m-match` | `.pf-c-log-viewer__list-item` | Indicates a list item is a search result. |
-| `.pf-m-current` | `.pf-c-log-viewer__list-item` | Indicates a list item is the current search result. |
+| `.pf-m-match` | `.pf-c-log-viewer__string` | Indicates a list item is a search result. |
+| `.pf-m-current` | `.pf-c-log-viewer__string` | Indicates a list item is the current search result. |
```
Comment (srambach) [inline src/patternfly/components/LogViewer/examples/LogViewer.md:127]: Should this say "Indicates the **string** is..."?

**Review by srambach — APPROVED** [review]
Looks good

**Pull request merged**